src/test/script_tests.cpp

```diff
@@ -940,7 +958,14 @@ BOOST_AUTO_TEST_CASE(script_json_test)
         std::string scriptSigString = test[pos++].get_str();
         CScript scriptSig = ParseScript(scriptSigString);
         std::string scriptPubKeyString = test[pos++].get_str();
-        CScript scriptPubKey = ParseScript(scriptPubKeyString);
+        CScript scriptPubKey;
+        // If requested, auto-generate the taproot output
+        if (strcmp(scriptPubKeyString.c_str(), "0x51 0x20 #TAPROOTOUTPUT#")== 0) {
+            BOOST_CHECK_MESSAGE(taprootBuilder.IsComplete(), "Failed to autogenerate Tapscript Script PubKey");
```

Comment:
Nit: I would use "taproot output key"  here instead of "Tapscript Script Pubkey" per [BIP341](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki)

Reply:
Fixed!